torch/_functorch/aot_autograd.py

```diff
@@ -1659,8 +1659,27 @@ def _collect_assigned_tensor_attributes(kp, v, _v):
                 # TODO(avik): Assigning all other types are allowed right now.
                 # Maybe in the future we want to limit this to primitive types?
 
+        new_attrs = _get_attributes(mod)
+        if len(new_attrs) != len(snapshot):
+            newly_created_attrs = []
+            deleted_attrs = []
+            for k in new_attrs:
+                if k not in snapshot:
+                    newly_created_attrs.append(k)
+
+            for k in snapshot:
+                if k not in new_attrs:
+                    deleted_attrs.append(k)
+
+            raise ValueError(
+                f"During export following attrs are deleted: {deleted_attrs} "
```

Comment:
Eh, we should tailor the error message when one of these is empty. Otherwise the message for the test is pretty weird (nothing was deleted). Frankly deleting would be pretty uncommon...

Also, say "the following attributes were" instead of "following attrs are"